openupgradelib/openupgrade.py

```diff
@@ -960,19 +962,35 @@ def reactivate_workflow_transitions(cr, transition_conditions):
             (condition, transition_id))
 
 
-def migrate(no_version=False):
+def migrate(no_version=False, v9env=None, context=None, *args, **kwargs):
     """
-    This is the decorator for the migrate() function
-    in migration scripts.
-    Set argument 'no_version' to True if the method as to be taken into account
-    if the module is installed during a migration.
-    Return when the 'version' argument is not defined and no_version is False,
-    and log execeptions.
-    Retrieve debug context data from the frame above for
-    logging purposes.
+    This is the decorator for the migrate() function in migration scripts.
+    It returns if `version` is not defined and `no_version` is False,
```

Comment:
I find this less clear than the original explanation of 'no_version'. Every function always 'returns'.